src/signal.cr

```diff
@@ -56,6 +56,7 @@ enum Signal : Int32
   USR1   = LibC::SIGUSR1
   USR2   = LibC::SIGUSR2
   WINCH  = LibC::SIGWINCH
+  STKSZ  = LibC::SIGSTKSZ
```

Comment:
Does this need to be exposed in Singal?

Reply:
I would guess probably not, unless there's a reason to expose stack size to users, but it's not clear to me whether we consider `LibC` bindings to be considered public API, and not clear whether `sigaltstack` and other functions for working with stacks would be relevant to most, if any, users. So I'll probably update this to avoid exposing `STKSZ` on `Signal`

Reply:
`LibC` bindings are not public API. They are there to be used by the standard library. Users can use `LibC`, but in most cases they shouldn't need to.

So yes, for now let's not add it to Signal's public API. We could always add it later if needed.

Reply:
great! addressed in 786c7d9